components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/datasource/QueryExecutor.java

```diff
@@ -39,9 +39,12 @@ public QueryExecutor() {
         queryManager = DataHolder.getInstance().getQueryManager();
     }
 
-    public boolean executeInsertQuery(String uuid, Blob businessRule, int deploymentStatus) throws BusinessRulesDatasourceException, SQLException {
+    public boolean executeInsertQuery(String uuid, byte[] businessRule, int deploymentStatus) throws
+            BusinessRulesDatasourceException, SQLException {
         Connection conn = dataSource.getConnection();
-        PreparedStatement statement = getInsertQuery(conn,uuid, businessRule, deploymentStatus);
+        Blob businessRuleBlob = conn.createBlob();
+        businessRuleBlob.setBytes(1,businessRule);
```

Comment:
There should be a space after the comma

Reply:
Fixed by bd9e014c11be3332f023a4e05e13149c5147ebb7